packages/fxa-payments-server/src/components/App/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { StripeProvider } from 'react-stripe-elements';
+import { Route, BrowserRouter } from 'react-router-dom';
+
+import './index.css';
+import Home from '../Home';
+
+export const App = ({ store, config }) => (
+  <StripeProvider apiKey={config.STRIPE_API_KEY}>
```

Comment:
Does the API key need to remain secret? Does it end up shipped with client side rendered JS?

Reply:
Talked about this a bit in Slack, but just to follow up: Yes, this ships out with rendered JS. But, this should be [a publishable API key](https://stripe.com/docs/keys) which Stripe describes like so:

>Publishable API keys are meant solely to identify your account with Stripe, they aren’t secret. In other words, they can safely be published in places like your Stripe.js JavaScript code, or in an Android or iPhone app. Publishable keys only have the power to create tokens.

Reply:
Oh, and for the sake of completion [re: tokens](https://stripe.com/docs/api/tokens):

> Tokenization is the process Stripe uses to collect sensitive card or bank account details, or personally identifiable information (PII), directly from your customers in a secure manner.